### ISSUE TITLE
Phasmid Studies

### ISSUE THREAD

**Issue by suwiding** [rdmpage/biostor#36]
@rdmpage I want to let you know that I got a bunch of citations from Phasmida Species File. I'm cleaning them up and expect to be done soon. When I finish, I'll upload an RIS file. BHL title http://www.biodiversitylibrary.org/bibliography/119312.

**Comment by rdmpage** [comment]
That's great, it will be nice to add that journal.

Sent from my iPhone




On Tue, Dec 13, 2016 at 2:38 AM +0000, "suwiding" <notifications@github.com> wrote:












@rdmpage I want to let you know that I got a bunch of citations from Phasmida Species File. I'm cleaning them up and expect to be done soon. When I finish, I'll upload an RIS file. BHL title http://www.biodiversitylibrary.org/bibliography/119312.

**Comment by suwiding** [comment]
@rdmpage 128 citations from Phasmid Studies are ready to be added to BioStor and BHL. The citations provide 100% coverage for the digitized issues currently in BHL.  I removed  2 citations for this publication that are already in BioStor and BHL.  The BHL start page number is in the notes field.  The database provider (DP) field is empty for these citations.
[phasmid_studies_to_biostor.txt.zip](https://github.com/rdmpage/biostor/files/678949/phasmid_studies_to_biostor.txt.zip)

**Comment by rdmpage** [comment]
@suwiding Nice work, I know how time consuming and tedious it can be to assemble these files. The articles are now in BioStor.

**Comment by suwiding** [comment]
@rdmpage I see the articles in BioStor when I search by author name, e.g. Bragg, but I don't see them when I go to http://biostor.org/issn/0966-0011.  Is it a matter of timing?  Will the articles show up after indexing is done?  I just noticed that the journal was in BHL without an ISSN and I added the print ISSN.  I also noticed that I gave the articles a journal name of 'Phasmid Studies' and that the BHL title record uses 'Phasmid studies' (different capitalization).  This won't cause trouble, will it?

**Comment by rdmpage** [comment]
@suwiding Whoops, it's a caching issue, try now e.g.
http://biostor.org/issn/0966-0011/year/2007

To try and make the BioStor web site faster and more reliable I've enabled
caching. If a page has been visited already (e.g., by Google's web crawler)
it won't be reloaded for 24 hours (I've now reduced this to 4 hours). One
consequence of this is that some recently added articles won't appear
straightaway.

BioStor groups articles into journals using the ISSN (or OCLC number if
there's no ISSN) so different journal spellings won't make a difference.
